### ISSUE TITLE
Create auto-sorted dataset to improve and accelerate labeling process

### ISSUE THREAD

**Issue by thenomemac** [matthew-sochor/fish.io.ai#70]
re-run best model with adam and 1e-4:1e-6 decay and 1e-3:1e-4 LR and save out all data sorted by cross entropy to rapidly improve labels

**Comment by thenomemac** [comment]
Via experimentation I foundout that sorting the images in each fish's folder by top 1 probability works way better than crossentropy for quickly finding likely miss bucketed images.

Images sorted with this algo can be found at `s3;//what-is-my-fish-images/catone`

**Comment by thenomemac** [comment]
Hoping to have all images filtered for a second pass through by tonight 4/23/17

**Comment by thenomemac** [comment]
Okay, you should use data set "trainfilter" "testfilter" when modeling from here on out for best results. Note, I have to rename all of the images in "trainfilter" s3 folder since all the images have an extra number appended to the front of the file name that'll mess up our data pipeline.

**Comment by matthew-sochor** [comment]
Great jorb!  This dramatically improved the modeling

**Comment by thenomemac** [comment]
for the record sorting by top K=1 prob worked better than cross entropy
